helm-charts/landing/templates/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Values.image.landing.name }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.landing.name }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - containerPort: {{ .Values.service.landing.internalPort }}
+          livenessProbe:
+            httpGet:
+              path: /
+              port: {{ .Values.service.landing.internalPort }}
+          readinessProbe:
+            httpGet:
+              path: /
+              port: {{ .Values.service.landing.internalPort }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+        - name: {{ .Values.image.landingSlackin.name }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.landingSlackin.name }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: SLACK_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "fullname" . }}
+                  key: slack-api-token
+          ports:
+            - containerPort: {{ .Values.service.landingSlackin.internalPort }}
+          livenessProbe:
+            tcpSocket:
+              port: {{ .Values.service.landingSlackin.internalPort }}
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            tcpSocket:
+              port: {{ .Values.service.landingSlackin.internalPort }}
+            initialDelaySeconds: 5
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+        - name: {{ .Values.image.landingApi.name }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.landingApi.name }}:{{ .Values.image.tag }}"
```

Comment:
I almost forgot! Please make `.Values.image.tag` mandatory.

Reply:
I will create a new PR for this.